aws/aws-how-to/instances/launch-ubuntu-desktop.rst

```diff
@@ -1,78 +1,105 @@
-Launch an Ubuntu desktop on EC2
-===============================
+Launch and connect to an Ubuntu Mantic desktop EC2 instance
+===========================================================
 
-To create an Ubuntu desktop environment on your EC2 VM, you can use TightVNC as your remote desktop server and Remmina as your local desktop client.
+Use the xrdp server and Remmina to connect to an AWS EC2 instance of Ubuntu Mantis.
 
-.. Note::
+Select and configure Ubuntu Mantic 
```

Comment:
The description in this section & in the next section needs to be more detailed, especially because there are quite a few screen changes happening during the process. Although we assume that the reader is familiar with the system, while giving instructions we still have to ensure that all steps are mentioned (as precisely as possible) and there is minimum room for confusion. So we need to stick to a specific flow and mention options/choices as and when required within the flow. 

We can use screenshots if something is not obvious or is difficult to find, but in general we prefer to minimise their use.